integration-tests/data/allTestsCases.ts

```diff
@@ -1345,7 +1347,7 @@ export const commonCases: TestCase[] = [
 ];
 
 export const kathmanduCases: TestCase[] = [
-  ...extractOp(150, 151, opMapping).map((op): TestCase => {
+  ...extractOp(150, 151, opMappingProto14).map((op): TestCase => {
```

Comment:
Can we move the following tests:
- Origination of a contract that contains the instructions EMIT
- Increase Paid Storage operation
into `commonCases` ? Those operation/instruction are now supported on every protocol we are testing against (Kathmandu, Lima, alpha and upcoming)